packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.test.tsx

```diff
@@ -180,6 +180,40 @@ describe('CommandBar', () => {
     expect(document.querySelector('.SubMenuClass')).toBeFalsy();
   });
 
+  it('passes overflowButton menuProps to the menu without changing current items', () => {
+    const items = [
+      {
+        name: 'TestText 1',
+        key: 'TestKey1'
+      }
+    ];
+
+    const overFlowItems = [
+      {
+        name: 'TestText 2',
+        key: 'TestKey2'
+      }
+    ];
+
+    const commandBar = mount(
+      <CommandBar
+        overflowButtonProps={{
+          menuProps: { items: [{ name: 'TestText 3', key: 'TestKey3' }], className: 'menuClassnameTest' }
+          // Items passed in should be ignored
+        }}
+        overflowItems={overFlowItems}
+        items={items}
+      />
+    );
+
+    const overflowMenuButton = commandBar.find('.ms-CommandBar-overflowButton');
+
+    overflowMenuButton.hostNodes().simulate('click');
+
+    expect(document.querySelectorAll('.ms-ContextualMenu-item')).toHaveLength(1);
+    expect(document.querySelector('.menuClassnameTest')).toBeTruthy();
```

Comment:
I'm a bit confused by this test. Which items should actually be present?  The comment above seems to say items passed in (like `menuClassNameTest`) should be ignored, but this is testing that it's present. Should it be present? It would be good to explicitly check for the items required: Item 1 and 2 in this case?

Reply:
the menuProps should be passed in (including things like className) but any 'items' passed in should be ignored (as they are handled via the component). I can change the wording if that didn't make sense.

Reply:
Oh I see, the menu items, not all items. Shouldn't the test make sure that TestKey3 isn't present?

Reply:
(I actually had the same confusion but didn't want to appear too nitpicky :p)